sacctmgr: add Chinese translation (#7588)
tldr-pages/tldr: pages/linux/sacctmgr.md

`sacctmgr add account {{account_name}} cluster={{cluster_of_account}}`

> - Show details of user/association/cluster/account (in a spcific format):

Reviewer: ```suggestion
- Show details of user/association/cluster/account using a specific format:
```

> `sacctmgr show {{user/association/cluster/account}} [format="Accout%10" format="GrpTRES%30"]`

Reviewer: ```suggestion
`sacctmgr show {{user|association|cluster|account}} [format="Accout%10" format="GrpTRES%30"]`
```

> `sacctmgr show {{user|association|cluster|account}} [format="Accout%10" format="GrpTRES%30"]`

Reviewer: oh, we don't use `[` to mark optional stuff.
```suggestion
`sacctmgr show {{user|association|cluster|account}} format="Accout%10" format="GrpTRES%30"`
```
Is this a valid example with 2 format arguments?

Author: It is valid. I use such commands to format my output.
